File: codigos/1 - estrutura sequencial/1015 - Distancia Ente Dois Pontos/Main.py
import math

x1, y1 = map(float, input().split())
x2, y2 = map(float, input().split())

distancia: float = math.sqrt(math.pow(x2-x1, 2) + math.pow(y2-y1, 2))

print(f"{distancia:.4f}")

File: codigos/1 - estrutura sequencial/1015 - Distancia Ente Dois Pontos/Main.dart
import 'dart:io';
import 'dart:math';

List<double> processar_entrada(){
  return stdin.readLineSync()!.split(' ').map((x) => double.parse(x)).toList();
}

double calcular_distancia(List<double> e1, List<double> e2){
  return sqrt(pow(e2[0] - e1[0], 2) + pow(e2[1] - e1[1], 2));
}

main(){
  String distancia = calcular_distancia(processar_entrada(), processar_entrada()).toStringAsFixed(4);
  print("$distancia");
}